src/main/kotlin/lotto/model/LottoMachine.kt

```diff
@@ -0,0 +1,122 @@
+package lotto.model
+
+import lotto.service.AmountValidator
+import lotto.service.Constants
+import lotto.view.InputView
+import lotto.view.OutputView
+import lotto.service.Rank
+
+object LottoMachine {
+    fun start() {
+        val amount = getAmountInput()
+        val manualTicketCount = getManualTicketCount(amount)
+        val manualTickets = getManualTickets(manualTicketCount)
+        val automaticTickets = generateAutomaticTickets(amount, manualTicketCount)
+        val winningNumbers = InputView.inputWinningNumbers()
+        val bonusNumber = InputView.inputBonusNumber(winningNumbers)
+        calculateStats(automaticTickets + manualTickets, winningNumbers, bonusNumber)
+        OutputView.displayResults(winStats, calculateReturnRate(amount))
+    }
+
+    private fun getAmountInput(): Int {
+        while (true) {
+            try {
+                val amount = InputView.inputPurchaseAmount()
+                AmountValidator.validate(amount)
+                return amount
+            } catch (_: IllegalArgumentException) {
+                println("Enter a valid amount to buy tickets")
+            }
+        }
+    }
+
+    private fun getManualTicketCount(amount: Int): Int {
+        while (true) {
+            try {
+                val manualTicketCount = InputView.inputManualTicketCount()
+                ManualTicket.checkTicketCount(manualTicketCount, amount)
+                return manualTicketCount
+            } catch (_: IllegalArgumentException) {
+                println("Enter a valid manual ticket count")
+            }
+        }
+    }
+
+    private fun getManualTickets(manualTicketCount: Int): List<Ticket> {
+        while (true) {
+            try {
+                val manualTicketsString = InputView.inputManualTicketNumbers(manualTicketCount)
+                val manualTickets = ManualTicket.generateManualTickets(manualTicketsString)
+                return manualTickets
```

Comment:
```suggestion
                return ManualTicket.generateManualTickets(manualTicketsString)
```

When we return the value of a variable right after initializing it, we can skip the initialization part and return it right away.

Reply:
Thats a nice simplification. Thank you for the suggestion, I had completely missed that. I'll update the code.